PPI/load_stage.sql

```diff
@@ -22,8 +22,8 @@ DO $_$
 BEGIN
 	PERFORM VOCABULARY_PACK.SetLatestUpdate( 
 	pVocabularyName			=> 'PPI',
-	pVocabularyDate			=> TO_DATE ('2018-12-28' ,'yyyy-mm-dd'), -- Date of Version Update from PPI Codebook
-	pVocabularyVersion		=> 'Codebook Version 0.3.34',  -- Current Codebook Version from PPI Codebook
+	pVocabularyDate			=> TO_DATE ('2020-05-07' ,'yyyy-mm-dd'), -- Date of Version Update from PPI Codebook
+	pVocabularyVersion		=> 'Codebook Version 0.3.34 + COVID + MHWB',  -- Current Codebook Version from PPI Codebook
```

Comment:
@Yevhenii-Bezeha 
Is 'COVID + MHWB' something to be specified here or in references?
Isn't it another vocabulary?

Reply:
@Alexdavv 
We spoke with PPI team before about this. And now it is the best way to show currently PPI has not only concepts from the Codebook Version 0.3.34, but also from the surveys COVID and MHWB (mental health well being). COVID - it is a short name from PPI's new survey.

Reply:
As far as I remember, the list of sources/references to the part of the terminology were never put to the vocabulary.vocabulary_version. Can you please check, BTW?

Why vocabulary_reference isn't working here?

Reply:
I don't know, unfortunately. If you have some ideas about it maybe we could find the problem.
This vocabulary_reference is there for a long time.